templates/include/glean/event.1.parquetmr.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "array",
+  "items": [
+    {
+      "type": "integer",
+      "minimum": 0
+    },
+    {
+      "type": "string"
+    },
+    {
+      "type": "string"
+    },
+    {
+      "type": [ "object", "null" ],
```

Comment:
Do we need `null` here?  If we don't have extra, this array will just have 3 items, right?

Reply:
This is a great idea. I like it.

Reply:
Cool.  So just `"type": "object"` here, then.